skimage/segmentation/_slic.pyx

```diff
@@ -200,18 +236,21 @@ def _slic_cython(np_floats[:, :, :, ::1] image_zyx,
 
 def _enforce_label_connectivity_cython(Py_ssize_t[:, :, ::1] segments,
                                        Py_ssize_t min_size,
-                                       Py_ssize_t max_size):
+                                       Py_ssize_t max_size,
+                                       Py_ssize_t start_label=1):
     """ Helper function to remove small disconnected regions from the labels
 
     Parameters
     ----------
-    segments : 3D array of int, shape (Z, Y, X)
```

Comment:
I think all these are wrong now.

Reply:
Do you mean the white space before the colons ` : `?

Reply:
Exactly.